src/sbvr-api/hooks.ts

```diff
@@ -49,35 +49,23 @@ export class SideEffectHook extends Hook {
 }
 
 // The execution order of rollback actions is unspecified
-const undoHooks = Promise.method(
-	(request: { hooks?: InstantiatedHooks<object> }) => {
-		if (request.hooks == null) {
-			return [];
-		}
-		return settleMapSeries(
-			_(request.hooks)
-				.flatMap()
-				.compact()
-				.value(),
-			hook => {
-				if (hook instanceof SideEffectHook) {
-					return hook.rollback();
-				}
-			},
-		);
-	},
-);
-
 export const rollbackRequestHooks = (
-	request:
-		| { hooks?: InstantiatedHooks<object> }
-		| Array<{ hooks?: InstantiatedHooks<object> }>,
+	hooks: InstantiatedHooks<object> | undefined,
```

Comment:
```suggestion
	hooks?: InstantiatedHooks<object>,
```

Reply:
I've done it this way because the parameter is not optional, it should always be provided or there's no point in calling the function, it's just it means that calling this as `rollbackRequestHooks(request.hooks)` becomes easier because `request.hooks` can be undefined

Reply:
Makes sense.